spec/models/expenses_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
```

Comment:
## Please Make sure the Unit and integrated test are all green.
- These bellows are the samples of your test I run locally 
![Capture](https://user-images.githubusercontent.com/63972670/189449427-cfcf89fd-ed53-4bcf-81c4-17b8e689b297.JPG)


- 
![Capture](https://user-images.githubusercontent.com/63972670/189449656-722a818f-bfc5-4ed4-87d9-91649eac1e62.JPG)

Reply:
![Screenshot from 2022-09-10 00-13-54](https://user-images.githubusercontent.com/97834160/189459473-90754a8f-c790-44a8-a601-09e6877fe6b8.png)

Okay :+1:  let me take another look even though on my side all the tests are runing well.